opentreemap/treemap/audit.py

```diff
@@ -743,12 +743,25 @@ def user_can_delete(self, user):
         field permissions on a model.
         """
 
-        is_admin = user.get_role(self.instance).name == Role.ADMINISTRATOR
+        is_admin = user.get_instance_user(self.instance).admin
```

Comment:
My mistake. This should be reverted. Better to always allow Administrators to delete than to always allow people with management access to delete.